readme.md

```diff
@@ -11,28 +11,27 @@
 For [Leiningen](https://github.com/technomancy/leiningen):
 
 ```clojure
```

Comment:
If you replace this clojure formatted version block with:
`[![Clojars Project](https://img.shields.io/clojars/v/bigml/clj-bigml.svg)](https://clojars.org/bigml/clj-bigml)`

Then the version badge will automatically update in the readme whenever we have a new release.